docs/csharp/structs.md

```diff
@@ -9,43 +9,46 @@ ms.assetid: a7094b8c-7229-4b6f-82fc-824d0ea0ec40
-  
-- Structs are value types and classes are reference types.  
-  
-- Unlike classes, structs can be instantiated without using a `new` operator.  
-  
-- Structs can declare constructors that have parameters.  
-  
-- A struct cannot inherit from another struct or class, and it cannot be the base of a class. All structs inherit directly from <xref:System.ValueType>, which inherits from <xref:System.Object>.  
-  
-- A struct can implement interfaces.
+
+But you declare and assign values to them as if they were simple non-aggregate types:
+
+[!code-csharp[Assign Values](../../samples/snippets/csharp/concepts/structs/assign-value.cs)]
+
+Value types are *sealed*, which means, for example, that you cannot derive a type from <xref:System.Int32>, and you cannot define a struct to inherit from any user-defined class or struct because a struct can only inherit from <xref:System.ValueType>. However, a struct can implement one or more interfaces. You can cast a struct type to an interface type; this causes a *boxing* operation to wrap the struct inside a reference type object on the managed heap. Boxing operations occur when you pass a value type to a method that takes an <xref:System.Object> as an input parameter. For more information, see [Boxing and Unboxing](./programming-guide/types/boxing-and-unboxing.md ).
+
+You use the [struct](./language-reference/keywords/struct.md) keyword to create your own custom value types. Typically, a struct is used as a container for a small set of related variables, as shown in the following example:
+
+[!code-csharp[Struct Keyword](../../samples/snippets/csharp/concepts/structs/struct-keyword.cs)]
+
+For more information about value types in the .NET Framework, see [Common Type System](../standard/common-type-system.md).
+
+Structs share most of the same syntax as classes, although structs are more limited than classes:
+
+- Within a struct declaration, fields cannot be initialized unless they are declared as `const` or `static`.
+
+- A struct cannot declare a parameterless constructor (a constructor without parameters) or a finalizer.
+
+- Structs are copied on assignment. When a struct is assigned to a new variable, all the data is copied, and any modification to the new copy does not change the data for the original copy. This is important to remember when working with collections of value types such as Dictionary<string, myStruct>.
+
+- Structs are value types and classes are reference types.
+
+- Unlike classes, structs can be instantiated without using a `new` operator.
+
+   > [!NOTE]
+   > In .NET Core 2.1 and later, structs must be instantiated by using a `new` operator. For more information, see [Breaking changes for migration from version 2.0 to 2.1](../core/compatibility/2.0-2.1.md#corefx).
```

Comment:
Is there any change for .NET Framework that we need to note?

Reply:
> In .NET Core 2.1 and later, structs must be instantiated by using a `new` operator

This is not quite correct. A struct must be instantiated by either `new`, `default` or by initializing all of the public fields. The latter is the rare case though, bordering on advanced level.